.github/workflows/_linux_transformers.yml

```diff
@@ -0,0 +1,173 @@
+name: Linux Transformers Test
+
+on:
+  workflow_call:
+    inputs:
+      pytorch:
+        required: false
+        type: string
+        default: 'nightly'
+        description: Pytorch branch/commit
+      python:
+        required: false
+        type: string
+        default: '3.10'
+        description: Python version
+      runner:
+        required: true
+        type: string
+        default: 'linux.idc.xpu'
+        description: Runner label
+      driver:
+        required: false
+        type: string
+        default: 'lts'
+        description: Driver lts/rolling
+      nightly_whl:
+        required: false
+        type: string
+        default: ''
+        description: Pytorch nightly wheel version
+      transformers:
+        required: false
+        type: string
+        default: 'v4.47.0'
+        description: Transformers version
+      transformers_test:
```

Comment:
Drop this. Let's focus on a single test - `backbone`.

Reply:
Please, drop `transformers_test`. We will most likely rework how to setup test scope.